src/DynamicData/List/ListEx.cs

```diff
@@ -62,7 +62,24 @@ public static void Clone<T>(this IList<T> source, IChangeSet<T> changes)
         /// changes
         /// </exception>
         public static void Clone<T>(this IList<T> source, IChangeSet<T> changes, IEqualityComparer<T> equalityComparer)
-        {
+        {
+            Clone(source, (IEnumerable<Change<T>>)changes, equalityComparer);
+        }
+
+        /// <summary> 
+        /// Clones the list from the specified enumerable of changes 
+        /// </summary> 
+        /// <typeparam name="T"></typeparam> 
+        /// <param name="source">The source.</param> 
+        /// <param name="changes">The changes.</param> 
+        /// <param name="equalityComparer">An equality comparer to match items in the changes.</param> 
+        /// <exception cref="System.ArgumentNullException"> 
+        /// source 
+        /// or 
+        /// changes 
+        /// </exception> 
+        public static void Clone<T>(this IList<T> source, IEnumerable<Change<T>> changes, IEqualityComparer<T> equalityComparer)

```

Comment:
Added to allow changes like the one found on TransformMany to clone without needing to create an intermediary ChangeSet